package NPersonalCards;

message TDismissCardRequest {
    required string  card_id = 1;

    message TAuth {
        optional string uid = 1; // Not required because uid can be in TVM user-ticket
    }
    optional TAuth auth = 2; // TODO(ndnuriev): Remove after PERSONALCARDS-24

    message TAppMetrika {
        optional string device_id = 1;
        optional string did = 2;
        optional string uuid = 3;
    }
    optional TAppMetrika app_metrika = 3; // TODO(ndnuriev): Remove after PERSONALCARDS-24

    optional uint64 yandexuid = 4; // TODO(ndnuriev): Remove after PERSONALCARDS-24
}

message TPushCard {
    required string card_id = 1;
    optional string tag = 2;
    optional string type = 3;
    optional uint32 date_from = 4;
    required uint32 date_to = 5;
}
